snippets/javascript/addeventlistener.md

```diff
@@ -0,0 +1,36 @@
+# addEventListener snippet in js
+
+## description
+
+> the addEventListener 'listens' to specific actions by the user and triggers a
+> functions when the action occurs
+
+## application
+
+> addEventListener allows the user to interact with a webpage in the example
+> below this is done through clicking a button
+
+## code
+
+```html
+<!-- create button on page body. use id or class to name it -->
+<button type="submit" id="user__btn">Start</button>
+```
+
+```javascript
+// declare and assign variable for button. use querySelector, getElementById
+let userBtn = document.querySelector("#user__btn");
```

Comment:
`const` is better here, since the value isn't going to change 🙂

Here is a snippet from [Wes Bos' Javascript course](https://wesbos.com/javascript/01-the-basics/variables-and-statements#scoping) explaining on when to use let or const:
![image](https://user-images.githubusercontent.com/4454851/145970963-537e0fbc-4c20-4675-945e-72349a70d190.png)

Reply:
edited